Fix handling of multiple entrez IDs, log warning

EszterToldi opened the pull request:
fixes #205

mtiberti commented on cancermuts/datasources.py:
```diff
@@ -232,8 +232,10 @@ def _get_aliases(self, gene_id, to, fr='UniProtKB_AC-ID'):
+
             if len(responses) > 1:
-                raise TypeError
+            	self.log.warning(f"Multiple {t} found for {gene_id}. Entrez ID processing and mutation retrieval from cBioPortal will be skipped. Found {t}: {[response['to'] for response in responses]}")
```
can you please
* remove the sentence that says that cBioPortal will be skipped, since this is not in the right place for it
* can the responses be comma-separated in the warning? e.g. `asd, qwe`. You can `', '.join()` for that
* can you also state that no identifier will be assigned for that type? (e.g. no Entrez ID will be assigned)

mtiberti commented on cancermuts/datasources.py:
```diff
@@ -232,8 +232,10 @@ def _get_aliases(self, gene_id, to, fr='UniProtKB_AC-ID'):
             if len(responses) > 1:
-                raise TypeError
+            	self.log.warning(f"Multiple {t} found for {gene_id}. Entrez ID processing and mutation retrieval from cBioPortal will be skipped. Found {t}: {[response['to'] for response in responses]}")
+            	return None
```
I don't think just returning `None` here works; normally we return a dictionary of responses, one per `t` in the loop. I would just set that one to `None`, and let the others as they are since they might just be valid, and then return the `out` dictionary as we already do

EszterToldi commented:
I now removed the sentence about cBioPortal, made the responses in the warning comma-separated, and added a sentence that no identifier will be assigned. I also set t to none, so that it should return the out dictionary as you requested. Please check if further changes are needed
